nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -141,6 +141,8 @@ In addition to numerous new and updated packages, this release has the following
 
 - [ReGreet](https://github.com/rharish101/ReGreet), a clean and customizable greeter for greetd. Available as [programs.regreet](#opt-programs.regreet.enable).
 
+- [RESTinstnace](https://github.com/asyrjasalo/RESTinstance), robotframework library for REST interfaces.
```

Comment:
This Python module will not be part of 23.05. The next release will be 23.11.

Personally I don't think that users will benefit of a listing of all new Python modules which will be available but it's your call.

Reply:
Yes, that would be way to overkill.